mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/query/parsing/nativequery/NativeQueryParserTest.java

```diff
@@ -279,4 +280,21 @@ public void shouldFindeCountQueryWithLogicalOperatorNOT() {
 		assertThat( queryDescriptor.getProjection() ).isNull();
 		assertThat( queryDescriptor.getOrderBy() ).isNull();
 	}
+
+	@Test
+	@TestForIssue(jiraKey = "OGM-900")
+	public void shouldSupportDotInCollectionName() {
```

Comment:
Nice! Could you add one more test which actually fetches an entity from a dot-named table?